vehicle/delete.go

```diff
@@ -20,5 +22,25 @@ func NewDeleteHandler(store storage.Store, logger *zap.Logger) *DeleteHandler {
 }
 
 func (d *DeleteHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
-	http.Error(rw, "Not Implemented", http.StatusInternalServerError)
+	strId := r.PathValue("id")
+
+	id, err := strconv.ParseInt(strId, 10, 64)
+
+	if err != nil {
+		fmt.Println("An error has occured", err)
+		rw.WriteHeader(http.StatusNotFound)
+		return
+	}
+
+	test, err := d.store.Vehicle().Delete(r.Context(), id)
+	if err != nil {
+		fmt.Println("An error has occured", err)
```

Comment:
de même plutôt "Failed to delete vehicle"

Reply:
de même gestion des erreurs : http.Error(rw, "Failed to delete vehicle", http.StatusInternalServerError)